requirements-common.txt

```diff
@@ -19,3 +19,4 @@ sentencepiece
 tensorflow-datasets
 safetensors
 pillow
+einops
```

Comment:
Are we okay with adding a new dependency? Simplifies the rearranging process, is commonly used and we can add new Keras Ops for the same functionality to avoid having a new dep.

Reply:
As we'll be adding a Keras operation for rearrange, we'll refactor this later to remove `einops`

Reply:
Thanks! Please create an issue, tag yourself and reference it here.

Reply:
Created an issue - though I can't assign since I'm not a collaborator on core Keras (only on KerasHub): https://github.com/keras-team/keras/issues/20332